### ISSUE TITLE
Issue on page /notebooks/llm.html

### ISSUE THREAD

**Issue by amyradurkin** [DeepLearning-JupyterBook/deeplearning-jupyterbook.github.io#39]
First mention of softmax in 2. Network doesn't have explanation, but further down the softmax function has brief description: "function to convert logits to probabilities". I think a similar description in the first mention would be helpful in case readers are unfamiliar
